services/autoscaling/src/simcore_service_autoscaling/core/settings.py

```diff
@@ -14,21 +15,57 @@
 from .._meta import API_VERSION, API_VTAG, APP_NAME
 
 
-class AwsSettings(BaseCustomSettings):
-    AWS_KEY_NAME: str
-    AWS_DNS: str
+class EC2Settings(BaseCustomSettings):
+    EC2_ACCESS_KEY_ID: str
+    EC2_ENDPOINT: Optional[str] = Field(
+        default=None, description="do not define if using standard AWS"
+    )
+    EC2_REGION_NAME: str = "us-east-1"
+    EC2_SECRET_ACCESS_KEY: str
+
+
+class EC2InstancesSettings(BaseCustomSettings):
+    EC2_INSTANCES_ALLOWED_TYPES: tuple[str, ...] = Field(
+        ...,
+        description="Defines which EC2 instances are considered as candidates for new docker nodes",
+    )
+    EC2_INSTANCES_AMI_ID: str = Field(
+        ..., description="Defines the AMI ID used to initialize a new docker node"
+    )
+    EC2_INSTANCES_MAX_INSTANCES: int = Field(
+        10,
+        description="Defines the maximum number of instances the autoscaling app may create",
+    )
+    EC2_INSTANCES_SECURITY_GROUP_IDS: list[str] = Field(
+        ..., description="TO BE DEFINED"
+    )
+    EC2_INSTANCES_SUBNET_ID: str = Field(..., description="TO BE DEFINED")
+    EC2_INSTANCES_KEY_NAME: str = Field(
+        ...,
+        description="SSH key filename (without ext) to access the docker swarm manager",
+    )
+
+
+class NodesMonitoringSettings(BaseCustomSettings):
+    NODES_MONITORING_NODE_LABELS: list[str] = Field(
+        default_factory=list,
+        description="autoscaling will only monitor nodes with the given labels (if empty all nodes will be monitored), these labels will be added to the new created nodes by default",
+    )
 
-    AWS_ACCESS_KEY_ID: str
-    AWS_SECRET_ACCESS_KEY: str
-    AWS_REGION_NAME: str = "us-east-1"  # see https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html
+    NODES_MONITORING_SERVICE_LABELS: list[str] = Field(
+        default_factory=list,
+        description="autoscaling will only monitor services with the given labels (if empty all services will be monitored)",
+    )
 
-    # EC2 instance paramaters
-    AWS_SECURITY_GROUP_IDS: list[str]
-    AWS_SUBNET_ID: str
+    NODES_MONITORING_SERVICE_IMAGE_NAMES: list[str] = Field(
+        default_factory=list,
+        description="autoscaling will only monitor services with the given image names (if empty all services will be monitored)",
```

Comment:
so I am going to remove this option for now, as it's not implemented and I see it is not easy to list task by image name. since we do not need it now, I will not waste time with it.